lib/rules/__tests__/index.test.js

```diff
@@ -1,12 +1,30 @@
 'use strict';
 
+const fs = require('node:fs');
+const path = require('node:path');
+
 const rules = require('..');
 
-describe('rules', () => {
-	for (const [ruleName, rule] of Object.entries(rules)) {
-		test(`the "${ruleName}" rule has metadata`, () => {
-			expect(rule.meta).toBeTruthy();
-			expect(rule.meta.url).toMatch(/^https:\/\/stylelint\.io\/.+/);
-		});
-	}
+describe('all rules', () => {
+	test.each(Object.entries(rules))('"%s" should have metadata', (_name, rule) => {
```

Comment:
See the [Jest `test.each()` doc](https://jestjs.io/docs/api#testeachtablename-fn-timeout).